geomesa-arrow/geomesa-arrow-gt/src/test/scala/org/locationtech/geomesa/arrow/io/DeltaWriterTest.scala

```diff
@@ -40,23 +41,25 @@ class DeltaWriterTest extends Specification {
     ScalaSimpleFeature.create(lineSft, s"$i", name, team, age, weight, s"2017-02-03T00:0$i:01.000Z", geom)
   }
 
+  val ipcOpts = new IpcOption() // TODO test legacy opts
```

Comment:
Do we want to hold up this PR on those tests?  Or just :shipit:?

Reply:
i'm not sure how we'd test it, since we'd need a classpath with arrow 0.10... so i'd say shipit (after we test it in the real world a bit)